### DISCUSSION TITLE
status sshd

### DISCUSSION THREAD

**Discussion by averlon** [fail2ban/fail2ban#3561] [Q&A] [answered]
Hi,
I have had a look at the status of jail "sshd":
```
Status for the jail: sshd
|- Filter
|  |- Currently failed: 0
|  |- Total failed:     10
|  `- File list:        /var/log/auth.log
`- Actions
   |- Currently banned: 0
   |- Total banned:     2
   `- Banned IP list:
```
What does "Total banned" tell me if there are no IP-addresses in the list. I guess, because "Currently banned" = 0.

**Comment by sebres** [comment, marked as answer]
See https://github.com/fail2ban/fail2ban/discussions/3413#discussioncomment-4197103 answering that.
